### PR TITLE
[Finishes #116787575] use codeworkspacecontainer in AppView, fix some rtl stuff

### PR DESCRIPTION
**Pull request by Bjvanminnen**
This PR started as an attempt to use CodeWorkspaceContainer everywhere. While working on it, I realized I'd regressed RTL pretty badly (for which we are getting user reports).

In an ideal world, I would have one PR that does just the RTL stuff and another one that starts using the CodeWorkspaceContainer, but given that we have a live bug that this fixes, it seems more prudent to not take the time to try to separate them (even if it might be somewhat more confusing).

Some things still need to be done after this PR
- GameLabView uses CodeWorkspaceContainer
- Eyes test for RTL
- Test for https://studio.code.org/s/allthethings/stage/13/puzzle/3 if we dont already have one, which uniquely uses react-only
- Have noViz and rtl be props for CodeWorkspaceContainer

What I've tested:
I've been going through allthethings and trying at least one level on each stage on both English and an RTL language.

### CODE REVIEW

File: apps/style/common.scss [hunk 1]
```diff
@@ -385,6 +385,10 @@ html[dir='rtl'] #codeWorkspace:not(.applab) {
   line-height: 400px;
 }
 
+html[dir='rtl'] div#visualizationResizeBar {
+  right: 400px;
+}
+
```
Comment (Bjvanminnen) [inline apps/style/common.scss:391]: Based on the code, I think the resize bar might have always been broken on RTL

Comment (Bjvanminnen) [inline apps/style/common.scss:391]: On a similar note, at some point I'd like to replace all these 400px with a variable (that maybe also ends up in JS).

**Comment by islemaster** [comment]
LGTM

File: code-studio/src/js/react-only.js [hunk 2]
```diff
@@ -6,3 +6,4 @@
 
 window.React = require('react');
 window.ReactDOM = require('react-dom');
+window.Radium = require('radium');
```
Comment (Bjvanminnen) [inline code-studio/src/js/react-only.js:9]: react-only is loaded on embed_blocks pages (like https://studio.code.org/s/allthethings/stage/13/puzzle/3). The name is becoming slightly less accurate with the addition of Radium, but Radium is still pretty React related so I don't feel too badly.

Comment (islemaster) [inline code-studio/src/js/react-only.js:9]: No, the name seems appropriate.  We load `React` and `ReactDOM` and `Radium` but in all it's sort of one framework and its plugins.

**Pull request merged**